src/main/java/com/eleks/academy/who_am_i/core/impl/RandomGame.java

```diff
@@ -0,0 +1,201 @@
+package com.eleks.academy.who_am_i.core.impl;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.eleks.academy.who_am_i.core.Game;
+import com.eleks.academy.who_am_i.core.Player;
+import com.eleks.academy.who_am_i.core.Turn;
+
+public class RandomGame implements Game {
+
+	private Map<String, String> playersCharacter = new HashMap<>();
+	private List<Player> players = new ArrayList<>();
+	private List<String> availableCharacters;
+	private Turn currentTurn;
+
+	private final static String YES = "Yes";
+	private final static String NO = "No";
+
+	public RandomGame(List<String> availableCharacters) {
+		this.availableCharacters = new ArrayList<String>(availableCharacters);
+	}
+
+	static {

```

Comment:
Semantically not correct, because it will get printed even if game is not started